src/cfnlint/data/ExtendedSpecs/all/03_value_types.json

```diff
@@ -4,33 +4,7 @@
     "path": "/ValueTypes",
     "value": {
       "AvailabilityZone": {
-        "AllowedValues": [
-          "af-south-1a", "af-south-1b", "af-south-1c",
-          "ap-east-1a", "ap-east-1b", "ap-east-1c",
-          "ap-northeast-1a", "ap-northeast-1b", "ap-northeast-1c", "ap-northeast-1d",
-          "ap-northeast-2a", "ap-northeast-2b", "ap-northeast-2c", "ap-northeast-2d",
-          "ap-northeast-3a",
-          "ap-south-1a", "ap-south-1b", "ap-south-1c",
-          "ap-southeast-1a", "ap-southeast-1b", "ap-southeast-1c",
-          "ap-southeast-2a", "ap-southeast-2b", "ap-southeast-2c",
-          "ca-central-1a", "ca-central-1b", "ca-central-1d",
-          "cn-north-1a", "cn-north-1b",
-          "cn-northwest-1a", "cn-northwest-1b", "cn-northwest-1c",
-          "eu-central-1a", "eu-central-1b", "eu-central-1c",
-          "eu-north-1a", "eu-north-1b", "eu-north-1c",
-          "eu-south-1a", "eu-south-1b", "eu-south-1c",
-          "eu-west-1a", "eu-west-1b", "eu-west-1c",
-          "eu-west-2a", "eu-west-2b", "eu-west-2c",
-          "eu-west-3a", "eu-west-3b", "eu-west-3c",
-          "me-south-1a", "me-south-1b", "me-south-1c",
-          "sa-east-1a", "sa-east-1b", "sa-east-1c",
-          "us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d", "us-east-1e", "us-east-1f", "us-east-1-bos-1a", "us-east-1-iah-1a", "us-east-1-mia-1a",
-          "us-east-2a", "us-east-2b", "us-east-2c",
-          "us-gov-east-1a", "us-gov-east-1b", "us-gov-east-1c",
-          "us-gov-west-1a", "us-gov-west-1b", "us-gov-west-1c",
-          "us-west-1a", "us-west-1b", "us-west-1c",
-          "us-west-2a", "us-west-2b", "us-west-2c", "us-west-2d", "us-west-2-lax-1a", "us-west-2-lax-1b"
-        ],
+        "AllowedPatternRegex": "[a-z0-9-]+",
```

Comment:
I get the value in this I just wonder if there is a better regex for this? 
`^[a-z]{2}-[a-z0-9-]+-[0-9][a-z]$`

Reply:
open to suggestions, that specific pattern would miss [wavelength zones](https://aws.amazon.com/wavelength/features/#Locations) though

Reply:
Was considering these two stricter patterns as well:
`[a-z]{2}-[a-z0-9-]+`
`(af|ap|ca|cn|eu|me|sa|us)-[a-z0-9-]+`
but wasn't sure if those patterns might eventually change

Reply:
Lets do what you have we can always improve it later.